### PR TITLE
fast5 0.6.5, nanopolish 0.9

### PR DESCRIPTION
**Pull request by nickloman**
* [*] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [*] This PR updates an existing recipe.

### CODE REVIEW

**Comment by bgruening** [comment]
@nickloman you need to increase the build number to get a new build.

**Comment by nickloman** [comment]
Hi @bgruening - Thanks! It seemed to build anyway, doesn't a version number change trigger a new build?

File: recipes/nanopolish/meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ requirements:
   build:
   - gcc
   - fast5 ==0.6.5
-  - htslib  {{ CONDA_HTSLIB }}*
+  - htslib >1.4
```
[the remark this answers is not in the document]
Comment (nickloman) [reply recipes/nanopolish/meta.yaml]: Nanopolish definitely needs 1.4 or higher to build, and I think I had a build with 1.3.

[the remark this answers is not in the document]
Comment (druvus) [reply recipes/nanopolish/meta.yaml]: Setting `htslib  {{ CONDA_HTSLIB }}*` will currently force it to use version 1.7.

**Comment by druvus** [comment]
@nickloman I bumped the Bioconda recipe to version 0.9.0 some time ago so your modification will not trigger a new build unless you increase the build number.

**Comment by druvus** [comment]
Closing this for now. Feel free to reopen it if needed.

**Pull request closed without merging**